main.tex

```diff
@@ -341,18 +728,125 @@ \subsubsection*{Mento Reserve Safety Fund}
     \item Voting rights: No.
 \end{itemize}
 
-\subsubsection{Additional Information About the MENTO Token}
+\section{Roadmap and Future Developments}
+\label{sec:roadmap}
+
+The Mento Platform's development path is structured around clear, actionable objectives aimed at broadening its financial ecosystem. Central to this expansion are the introduction of the MENTO token and the establishment of a decentralized governance structure.
+
+\begin{itemize}
+    \item \textbf{Governance Transition:}
+    The Mento Governance was activated on (July 2025) and the MENTO token was previously distributed according to the allocations described in Section \ref{sec:token_distribution}. This new structure now empowers the community to propose, debate, and implement changes directly through a democratic voting process. Initial governance actions will focus on community-led development and innovation. 
```

Comment:
reminder to replace (July 2025) with a concrete date or instead just say "was activated in July 2025"